### ISSUE TITLE
@twilio-paste/core 16.0.0 -- Cannot find module '@twilio-paste/button/src/styles' from 'node_modules/@twilio-paste/radio-button-group/dist/index.js'

### ISSUE THREAD

**Issue by btjfsu** [twilio-labs/paste#2756]
Thanks for helping make Paste better! For general questions or feature requests, please post [a discussion](https://github.com/twilio-labs/paste/discussions/new).

**Describe the bug**

While upgrading `@twilio-paste/core` from 15.3.0 to 16.0.0, we noticed where running jest tests that tests started failing due to a module not being found by the new package `@twilio-paste/radio-button-group`. 

**To Reproduce**

1. Updated `@twilio-paste/core` from 15.3.0 to 16.0.0 in package.json 
2. Installed NPM packages
3. Run unit tests (e.g. Jest)
5. See the following error (example below)
``` 
  ● Test suite failed to run

    Cannot find module '@twilio-paste/button/src/styles' from 'node_modules/@twilio-paste/radio-button-group/dist/index.js'

    Require stack:
      node_modules/@twilio-paste/radio-button-group/dist/index.js
      node_modules/@twilio-paste/core/dist/index.js
      src/shared/components/ConnectedAnchorButton/index.tsx
      src/shared/components/RequestInspector/components/RequestInspectorPopover/index.tsx
      src/shared/components/RequestInspector/components/RequestInspectorPopover/index.test.tsx

```

**Comment by gloriliale** [comment]
hi @btjfsu! Thank you for bringing this to our attention- we're working rapidly to resolve it and will update here with the fix soon!

**Comment by btjfsu** [comment]
Looks like @SiTaggart already has a PR lined up for this one --> #2757

**Comment by btjfsu** [comment]
I can confirm that the fixed (#2757) introduced in `@twilio-paste/core` 16.0.1 resolves the issue above. Thank you for addressing this issue so quickly @gloriliale and @SiTaggart!!! Closing this issue.
